fdbutil/filter_fdb_entries.py

```diff
@@ -87,15 +87,16 @@ def filter_fdb_entries(fdb_filename, arp_filename, config_db_filename, backup_fi
         Returns:
             None
     """
-    arp_map = get_arp_entries_map(arp_filename, config_db_filename)
+    arp_map, switch_mac = get_arp_entries_map_and_switch_mac(arp_filename, config_db_filename)
```

Comment:
since switch_mac is used only in this function, can we have it as a separate API to fetch instead of changing multiple existing APIs?

Reply:
It could, I just did not want to open the json files twice.